--- setup.py.bak	2012-06-25 19:37:52.958036279 -0400
+++ setup.py	2012-06-25 19:41:38.370746630 -0400
@@ -372,6 +372,7 @@
                              'pandas/src/datetime/np_datetime.c',
                              'pandas/src/datetime/np_datetime_strings.c'],
                     include_dirs=[np.get_include()],
+                    libraries=['m']
                     # pyrex_gdb=True,
                     # extra_compile_args=['-Wconversion']
                     )
@@ -382,12 +383,14 @@
                        sources=[srcpath('plib', suffix=suffix),
                                 'pandas/src/datetime/np_datetime.c',
                                 'pandas/src/period.c'],
-                       include_dirs=[np.get_include()])
+                       include_dirs=[np.get_include()],
+                       libraries=['m'])
 
 
 sparse_ext = Extension('pandas._sparse',
                        sources=[srcpath('sparse', suffix=suffix)],
-                       include_dirs=[np.get_include()])
+                       include_dirs=[np.get_include()],
+                       libraries=['m'])
 
 sandbox_ext = Extension('pandas._sandbox',
                         sources=[srcpath('sandbox', suffix=suffix)],
