--- setup.py.orig	2014-02-09 14:31:48.000000000 -0500
+++ setup.py	2014-03-04 12:33:38.570228597 -0500
@@ -458,7 +458,8 @@
     obj = Extension('pandas.%s' % name,
                     sources=sources,
                     depends=data.get('depends', []),
-                    include_dirs=include)
+                    include_dirs=include,
+                    libraries=['m'])
 
     extensions.append(obj)
 
@@ -518,7 +519,8 @@
                       include_dirs=['pandas/src/ujson/python',
                                     'pandas/src/ujson/lib',
                                     'pandas/src/datetime'] + common_include,
-                      extra_compile_args=['-D_GNU_SOURCE'])
+                      extra_compile_args=['-D_GNU_SOURCE'],
+                      libraries=['m'])
 
 
 extensions.append(ujson_ext)
